metric_config_parser/tests/integration/test_config_integration.py

```diff
@@ -4,16 +4,9 @@
 class TestConfigIntegration:
     def test_configs_from_repo(self):
         config_collection = ConfigCollection.from_github_repos(
-            ["https://github.com/mozilla/metric-hub", "https://github.com/mozilla/jetstream-config"]
+            ["https://github.com/mozilla/metric-hub"]
         )
         assert config_collection is not None
-        assert config_collection.get_platform_defaults("firefox_desktop") is not None
-        assert (
-            config_collection.spec_for_outcome(
-                config_collection.outcomes[0].slug, config_collection.outcomes[0].platform
-            )
-            is not None
-        )
```

Comment:
Any reason to remove these assertions? If they are not defined then we could change the assertion to `is None`.